src/tools/m65common.c

```diff
@@ -1597,6 +1609,8 @@ void open_the_serial_port(char* serial_port)
   fd.fdfile = open_serial_port(serial_port, 2000000);
   if (fd.fdfile == INVALID_HANDLE_VALUE) {
     fprintf(stderr, "Could not open serial port '%s'\n", serial_port);
+    fprintf(stderr, "  (could the port be in use by another application?)\n");
```

Comment:
If its a TCP port in use, you could use netstat or similar to display some info, but don't worry about it for now.  More generally, lsof can give you information of use here for serial ports, too.

Reply:
No probs, I don't mind the prog getting more smarts on why the connection fails in future.

The current addition here came about due to a suggestion from deft that we hint at a few common possibilities for the user to consider and confirm on their end, rather than bailing out without much of a clue for them on why.